initial_setup.py

```
@@ -0,0 +1,667 @@
+#!/usr/bin/env python3
+"""Initial setup script for Glimpser application.
+
+This script creates an admin user, configures sane default settings,
+and adds example templates to help new users get started.
+"""
+
+import argparse
+import getpass
```

Ruff (0.11.9)

`getpass` imported but unused

Remove unused import: `getpass`

(F401)